packages/react/src/components/header/components/navigationLink/NavigationLink.tsx

```diff
@@ -191,6 +196,7 @@ export const NavigationLink = <T extends React.ElementType = 'a'>({
       }
     }
   };
+  console.log(dropdownButtonClassName);
```

Comment:
I'm guessing this was left here unintentionally? 🤔

Reply:
Yes 🤦 Thanks for finding this!